auth_saml_create_user/models/auth_saml.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2010-2016 XCG Consulting <http://odoo.consulting>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
why this file has different copyright and license

Reply:
Done, thanks